### PR TITLE
Adds font color to due period depending on severity.

### LINKED ISSUE #116: Home page and Details page call GetDueClass
1. On the **home** page, where we currently display the text "x days", "x days ago", "today" etc... we should colorize that text by calling `CssUtil.GetDueClass` inside the html div/container that contains thoose values
2. We should do the same on the **details** page but instead we want to color the whole div that contains the next payment date, including it's label. This should make it hard to miss that a payment is overdue or due soon

### PR DESCRIPTION
**Pull request by denislavh**
I am not sure if the details page should have the font changed or the background. Closes #116.

### CODE REVIEW

File: subtrack.MAUI/Pages/Index.razor [hunk 1]
```diff
@@ -36,7 +36,7 @@
                             <i class="fa fa-repeat" aria-hidden="true"></i>
                         }
                     </div>
-                    <div class="text-end">@GetDueDaysText(sub.DueDays)</div>
+                    <div class=@CssUtil.GetDueClass(sub.DueDays)>@GetDueDaysText(sub.DueDays)</div>
```
Comment (chrisK00) [inline subtrack.MAUI/Pages/Index.razor]: Did you try running this? I think we still need the text-end because it aligns the text

**Review by chrisK00 — CHANGES_REQUESTED** [review]
Just need to color the text inside the div so the current implementation is correct

File: subtrack.MAUI/Pages/Index.razor (hunk 1 above)
Comment (denislavh) [reply subtrack.MAUI/Pages/Index.razor]: home
detail
I am not sure why but it seems to have retained the position of the text.

Comment (chrisK00) [reply subtrack.MAUI/Pages/Index.razor]: > home
> detail
> I am not sure why but it seems to have retained the position of the text.

Could be bug related. Before you remove it check the git history on that line, alternatively just keep it so you dont need to waste time

Comment (chrisK00) [reply subtrack.MAUI/Pages/Index.razor]: > home
> detail
> I am not sure why but it seems to have retained the position of the text.

Btw you should really pay your paramount subscription

**Review by chrisK00 — APPROVED** [review]
Gj!

**Pull request merged**